webhook.go

```diff
@@ -43,6 +43,8 @@ var (
 
 	watcher *fsnotify.Watcher
 	signals chan os.Signal
+
+	limits = make(map[string]chan struct{})
```

Comment:
For the sake of verbosity, I would rename `limits` to `hookExecutions`

Reply:
fixed